packages/nodes-base/nodes/Redis/Redis.node.ts

```diff
@@ -766,8 +766,7 @@ export class Redis implements INodeType {
 								}
 
 								for (const keyName of keys) {
-									// eslint-disable-next-line @typescript-eslint/await-thenable
-									item.json[keyName] = await promises[keyName];
+									item.json[keyName] = promises[keyName];
```

Comment:
Would be nice to rename the `promises` variable as well since they are no longer promises what it stores